### ISSUE TITLE
[feature request] Support for cuda 11

### ISSUE THREAD

**Issue by yzh119** [apache/incubator-tvm#6157]
Currently, if I build tvm with cublas and CUDA 11, I'll encounter the following error:
```
/home/***/repo/incubator-tvm/src/runtime/contrib/cublas/cublas.cc: In function ‘void tvm::contrib::CallLtIgemm(tvm::runtime::TVMArgs, tvm::runtime::TVMRetValue*, cublasLtHandle_t)’:
/home/***/repo/incubator-tvm/src/runtime/contrib/cublas/cublas.cc:175:63: error: cannot convert ‘cudaDataType_t’ to ‘cublasComputeType_t’
  175 |   CHECK_CUBLAS_ERROR(cublasLtMatmulDescCreate(&operationDesc, CUDA_R_32I));
      |                                                               ^~~~~~~~~~
      |                                                               |
      |                                                               cudaDataType_t
```

which is no longer compatible with CUDA 11 [cublasLtMatmulDescCreate interface](https://docs.nvidia.com/cuda/cublas/index.html#cublasLtMatmulDescCreate), the new API has three arguments:
```
cublasStatus_t cublasLtMatmulDescCreate(
	cublasLtMatmulDesc_t *matmulDesc, 
	cudaDataType computeType);
```
it breaks the old api which only has two arguments.

I'm not sure if there are other breaking changes in CUDA 11 that would fail the compilation of tvm. I think a macro like `#if CUDART_VERSION >= 11000` could solve the problem mentioned above.

**Comment by tqchen** [comment]
Thanks @yzh119 for bringing it up, would you like to take a stab? cc @Laurawly @icemelon9

**Comment by tqchen** [comment]
closed by #6213 thanks to @lanchongyizu
